app/src/main/java/org/systers/mentorship/view/activities/SplashScreenActivity.kt

```diff
@@ -14,12 +15,19 @@ class SplashScreenActivity : AppCompatActivity() {
     private lateinit var handler: Handler
     private lateinit var runnable: Runnable
     private var SPLASH_DISPLAY_LENGTH: Long = 1000
+    private val preferenceManager: PreferenceManager = PreferenceManager()
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_splash_screen)
 
-        val intent = Intent(this, LoginActivity::class.java)
+        val intent = if (preferenceManager.authToken == "") {
+            Intent(this, LoginActivity::class.java)
+        }
+
```

Comment:
remove this line and have like this just to keep consistency across other if elses:
```
(...)

} else {

(...)
```